### PR TITLE
Channels sections in Client Cfg & Admin

### PR DESCRIPTION
**Pull request by Loquacity**
(This relates to https://bugzilla.novell.com/show_bug.cgi?id=1124634 but is broader than just that bug.)

* Added Channels section to client config, cribbed from reference-webui-channels.adoc, with reference to Admin Guide
* Added Channels section to Admin Guide, with outline for the section, and content from BZ1124634

Todo:

- [ ] Complete writing sections in Admin Guide

### CODE REVIEW

File: modules/administration/pages/channel-management.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+TODO:: write this
+
+== Delete Custom Channels
+
+TODO:: write this
+
+Note that when channels are deleted, the packages that are part of the deleted channel are not automatically removed.
+If you try to update packages that have had their channel deleted, it can cause errors.
+In this case, you will need to manually remove the packages.
```
Comment (lkotek) [inline modules/administration/pages/channel-management.adoc]: I would mention it's necessary to set those package to "removed" in package state page. (States > Package on the system page.) Simple removal of the package by zypper or thru "Software" menu doesn't solve the problem as package installation is still enforced by state.

File: modules/administration/pages/channel-management.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+include::ROOT:partial$entities.adoc[]
+endif::[]
+// PDF entities
+ifdef::backend-pdf[]
+include::_partials/entities.adoc[]
+endif::[]
+
+Channels are a method of grouping software packages.
+
+In {productname}, channels are grouped into base and child channels, with base channels usually grouped by hardware architecture and software version, and child channels being related to a specific base channel.
```
Comment (mcalmer) [inline modules/administration/pages/channel-management.adoc]: The base channel defines the OS (type, version and arch) while the child channels should be compatible to it.
When a system later gets assigned to a base channel, it is limited to only select the child channels of its base channel.

The relation base <=> child is all about compatibility. You should not use a SLES10 SP1 s390x base channel and define a Ubuntu universe 18.04 child channel below it.

File: modules/client-configuration/pages/channels.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+There are two types of software channels: base channels and child channels.
+
+Base Channels::
+A base channel consists of packages built for a specific architecture and release.
+For example, all of the packages in {sls}{nbsp}12 for the `x86_64` architecture make up a base channel.
+The list of packages in {sls}{nbsp}12 for the `s390x` architecture make up a different base channel.
++
+
+A system must be subscribed to only one base channel assigned automatically during registration based on the {sle} release and system architecture.
+For paid base channels, an associated subscription must exist.
```
Comment (mcalmer) [inline modules/client-configuration/pages/channels.adoc]: I think we use the term "Vendor Channels". Not sure what is better.

File: modules/client-configuration/pages/channels.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Child Channels::
+A child channel is associated with a base channel and provides extra packages.
+For example, an organization can create a child channel associated with {sls} on `x86_64` architecture that contains extra packages for a custom application.
```
Comment (mcalmer) [inline modules/client-configuration/pages/channels.adoc]: This statement sounds more like the difference between Vendor and Custom Channels.

Please remember:

At SUSE we have always a `Pool` and an `Update` channel. The Pool defines in most cases the base channel while the Update channel is already a child. And with SLE15 the situation is more scary. The Base Channel just contains 3 packages while the `SLE-Module-Basesystem` (Pool and Update - both childs) contains the important parts of the packages.

**Comment by Loquacity** [comment]
> @Loquacity In the Reference guide, we need a reference to the Admin guide where the reader can find more about channels.

Agreed, but I think that's for after @jcayuoutte has finished migrating it.

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Agreed, but I think that's for after @jcayuoutte has finished migrating it.

Let's hope we will not forget about it.

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

File: modules/administration/pages/channel-management.adoc [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+{productname} Administrators and Channel Administrators have channel management authority, which gives them the ability to create and manage their own custom channels.
+
+This section gives more detail on how to create, administer, and delete custom channels.
+
+== Create Custom Channels
+
+If you have custom software packages that you need to install on your {productname} systems, you can create a custom child channel to manage them.
+You will need to create the channel in the {productname} {webui} and create a repository for the packages, before assigning the channel to your systems.
+
+Note that if you include a package tagged with [systemitem]``release`` in your channel, then it will be automatically installed when the channel is installed.
```
Comment (Bischoff) [inline modules/administration/pages/channel-management.adoc]: `-release` at the end of package name (note the dash)

File: modules/administration/pages/channel-management.adoc [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+You can hold the btn:[Ctrl] key to select multiple channels, if you need to.
+. Click btn:[Update Activation Key].
+
+
+
+== Delete Custom Channels
+
+TODO:: write this
+
+Note that when channels are deleted, the packages that are part of the deleted channel are not automatically removed.
```
Comment (Bischoff) [inline modules/administration/pages/channel-management.adoc]: cool, thanks

**Comment by Loquacity** [comment]
> > @Loquacity In the Reference guide, we need a reference to the Admin guide where the reader can find more about channels.
> 
> Agreed, but I think that's for after @jcayouette has finished migrating it.

I'll create another issue if the Admin Guide isn't ready by the time I complete this.

**Comment by Loquacity** [comment]
> Lana Brindley <notifications@github.com> writes:
> Agreed, but I think that's for after @jcayuoutte has finished migrating it.
> Let's hope we will not forget about it.

I'll create another issue if the Admin Guide isn't ready by the time I complete this.

**Comment by Loquacity** [comment]
Created https://github.com/SUSE/spacewalk/issues/7312 to track Ref Guide changes. Merging this one.

**Pull request merged**